pkg/api/v1/types.go

```diff
@@ -557,6 +561,10 @@ type PersistentVolumeClaimSpec struct {
 	// VolumeName is the binding reference to the PersistentVolume backing this claim.
 	// +optional
 	VolumeName string `json:"volumeName,omitempty" protobuf:"bytes,3,opt,name=volumeName"`
+	// Name of the StorageClass where the desired volume belongs to.
+	// More info: http://kubernetes.io/docs/user-guide/persistent-volumes#class-1
+	// +optional
+	ClassName *string `json:"className,omitempty"`
```

Comment:
Because when PV.Spec.Class is missing it will default to empty string and hence it will be always filled. When PVC.Spec.Class is missing it means something else that empty string and we need to distinguish between these two cases.